### ISSUE TITLE
Sentence field in NameTable and EventTable is ML

### ISSUE THREAD

**Issue by optout2024** [ricko2001/RootsMagic_Database_Design#1]
It looks like the 'Sentence' Field in both the NameTable and EventTable supports multi-line text. I don't expect that there is much of a need for this but it is supported by the db

**Comment by ricko2001** [comment]
Interesting. I never tried putting a new line in the sentence template, but it works.
Fields in SQLite of type TEXT will always allow multiple lines. I was trying to indicate what kinds of data RM expects.

Do you think it's helpful to try to distinguish these fields ?
In RM, look at any Fact, most have a Description field and all have a Note field. They are both saved in a SQLite TEXT column, yet the GUI handles them differently.

**Comment by optout2024** [comment]
Agreed that a multi-line sentence is not typical. In the spreadsheet I ended up adding a cell note that ML is supported. The "typical" value links to the RM sentence formula wiki page. I just realized that I should update that link to the RM9 help page since the wiki is for RM8. 

I liked that your pages distinguish SL from ML.  I don't think fields like Event.Description support a new line.  It seemed worth noting in the Typical value column and I started to updated the spreadsheet accordingly. I'm interested if you're rethinking that. In fact, I was wondering if it was worth noting which fields are supported by the Note Editor because of the addition features that it provides.
